src/app/services/film/findAll.js

```diff
@@ -10,6 +10,10 @@ module.exports = (page = 1, limit = 10, filters) => {
       [Op.iLike]: '%' + filters.name + '%'
     }
   }
+  if (filters.rating) {
```

Comment:
Como esse atributo é um número no model, seria bom validar se é diferente de undefined ou null, pois se for possível filtrar por 0 o if irá retornar false e não entrará no bloco dele

Reply:
Faz sentido... Mas esse filtro tá funcionando corretamente, mesmo com o valor 0.

Reply:
Alinhado pelo chat